platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -43,17 +43,21 @@ extension PolyOut {
         return documentDirectory.appendingPathComponent(PolyOut.fsFilesRoot).appendingPathComponent(activeFeature!.id)
     }
     
-    private func pathFromUrl(url: String) -> URL? {
-        let id = idFromUrl(url: url)
-
-        return id != nil ? featureFilesPath().appendingPathComponent(id!) : nil
-    }
+    /// There are 2 types of urls that are used
+    /// 1. Filesystem url that starts with file:// and points to a physical location on the device
+    /// 2. Polypod url that starts with polypod:// and is how features access their resources in a platform independent way
```

Comment:
Sorry, I don't understand very well the context for this comment. Is it about the next function?

Reply:
It is about the group of functions following it, related to mapping PolyPod urls to Filesystem urls.

Reply:
It would be better to try and find a better way to do it. One way that's used by some of us (notably @fhd) is to add a PR comment, so that it's clear for the reviewer, but does not leave a permanent footprint. Another option, not used by anyone, even more so in this Swift codebase, is to document the package, or class, or function. Since we don't have that, probably not the best way.

Reply:
I can remove it. It was for me to clarify what is happening.